deploy/buildtools/bit_builders/vitis.py

```diff
@@ -0,0 +1,206 @@
+            return
+
+        # The default error-handling procedure. Send an email and teardown instance
+        def on_build_failure():
+            """ Terminate build host and notify user that build failed """
+
+            message_title = "FireSim Vitis FPGA Build Failed"
+
+            message_body = "Your FPGA build failed for triplet: " + self.build_config.get_chisel_triplet()
+
+            send_firesim_notification(message_title, message_body)
+
+            rootLogger.info(message_title)
+            rootLogger.info(message_body)
+
+            self.build_config.build_config_file.build_farm.release_build_host(self.build_config)
+
+        rootLogger.info("Building Vitis Bitstream from Verilog")
+
+        local_deploy_dir = get_deploy_dir()
+        fpga_build_postfix = f"cl_{self.build_config.get_chisel_triplet()}"
+        local_results_dir = f"{local_deploy_dir}/results-build/{self.build_config.get_build_dir_name()}"
+
+        build_farm = self.build_config.build_config_file.build_farm
+
+        # 'cl_dir' holds the eventual directory in which vivado will run.
+        cl_dir = self.cl_dir_setup(self.build_config.get_chisel_triplet(), build_farm.get_build_host(self.build_config).dest_build_dir)
+
+        # TODO: Does this still apply or is this done in the Makefile
+        ## copy over generated RTL into local CL_DIR before remote
+        #with InfoStreamLogger('stdout'), InfoStreamLogger('stderr'):
+        #    run("""mkdir -p {}""".format(local_results_dir))
+        #    run("""cp {}/design/FireSim-generated.sv {}/FireSim-generated.sv""".format(cl_dir, local_results_dir))
+
+        vitis_result = 0
+        with InfoStreamLogger('stdout'), InfoStreamLogger('stderr'):
+            # TODO: Put script within Vitis area
+            # copy script to the cl_dir and execute
+            rsync_cap = rsync_project(
+                local_dir=f"{local_deploy_dir}/../platforms/vitis/build-bitstream.sh",
+                remote_dir=f"{cl_dir}/",
+                ssh_opts="-o StrictHostKeyChecking=no",
+                extra_opts="-l", capture=True)
+            rootLogger.debug(rsync_cap)
+            rootLogger.debug(rsync_cap.stderr)
+
+            vitis_result = run(f"{cl_dir}/build-bitstream.sh {cl_dir}").return_code
+
+        # put build results in the result-build area
+        with StreamLogger('stdout'), StreamLogger('stderr'):
+            rsync_cap = rsync_project(
+                local_dir=f"{local_results_dir}/",
+                remote_dir=cl_dir,
+                ssh_opts="-o StrictHostKeyChecking=no", upload=False, extra_opts="-l",
+                capture=True)
+            rootLogger.debug(rsync_cap)
+            rootLogger.debug(rsync_cap.stderr)
+
+        if vitis_result != 0:
+            on_build_failure()
+            return
+
+        finame = self.build_config.name
+        xclbin_path = cl_dir + "/bitstream/build_dir.xilinx_u250_gen3x16_xdma_3_1_202020_1/firesim.xclbin"
```

Comment:
For this release lets try to change the build process so the shell name doesn't show up here.

Reply:
Sure. I'll change this when we flatten the Vitis submodule into the repo.